app/scss/_withUserForm.scss

```diff
@@ -38,6 +38,12 @@
     margin: 24px 0;
 }
 
+.userFormInputFieldLocationContainer {
+    display: flex;
+    width: 500px;
```

Comment:
Having a set width here will make the page not mobile responsive. If you want to have a set width for this container, you will have 
 media query that will handle mobile screens.

Reply:
![Screen Shot 2020-09-11 at 1 08 52 AM](https://user-images.githubusercontent.com/57198255/92864649-5cdb2d80-f3cb-11ea-81b9-0467ce27c697.png)
 
I addressed the issue with a media query to the end and set the width to a mobile-friendly length. Pushing changes now!